VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
We already have the version of vector-datasource specified in setup.py though. Can we just keep it in there?